### PR TITLE
[MIG] : Migrated sale_stock_picking_blocking to v13.

### PR DESCRIPTION
**Pull request by sudhir-erpharbor**

### CODE REVIEW

**Comment by rjpathan** [comment]
@rousseldenis please confirm when this PR will be merged.
Thanks

**Comment by rousseldenis** [comment]
> @rousseldenis please confirm when this PR will be merged.
> Thanks

You'll be notified as you are part of this conversation.

Can you make a review  ?

File: sale_stock_picking_blocking/views/res_partner_view.xml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  <!--(http://www.eficent.com)-->
+<!--License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).-->
+    <record id="view_partner_form" model="ir.ui.view">
+        <field name="name">res.partner.form</field>
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="base.view_partner_form"/>
+        <field name="arch" type="xml">
+            <group name="sale" position="inside">
+                    <field name="default_delivery_block"
+                        attrs="{'invisible': ['|', ('is_company', '=', False), ('parent_id', '!=', False)]}"/>
```
Comment (rjpathan) [inline sale_stock_picking_blocking/views/res_partner_view.xml]: I think **Default Delivery Block Reason** field should be visible for Individual Customer.

Comment (rjpathan) [reply sale_stock_picking_blocking/views/res_partner_view.xml]: @sudhir-erpharbor please fix if this makes sense, so this PR can be merged.

> I think **Default Delivery Block Reason** field should be visible for Individual Customer.

**Review by yung-wang — APPROVED** [review]
Tested and works fine here

**Review by danimv5 — COMMENTED** [review]
@sudhir-erpharbor  Everything works fine. But, when a contact is "individual", the field: "Default Delivery Block Reason" doesn't appear.

**Comment by sudhir-erpharbor** [comment]
@rjpathan @danimv5 

I Updated the code and now the field "Default Delivery Block Reason" will be displayed for the "Individual" partner also.

**Review by danimv5 — APPROVED** [review]
Perfect! Functional review. LGTM!

**Comment by gurneyalex** [comment]
@sudhir-erpharbor hello can you enable pre-commit and rebase your PR? this will help you make the CI green, and then we can proceed and merge this. Thanks :smile_cat:

**Comment by dreispt** [comment]
Superseded by https://github.com/OCA/sale-workflow/pull/1645

**Pull request closed without merging**